docker/kafka-setup/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Using as a base image because to get the needed jars for confluent utils
-FROM confluentinc/cp-base-new:6.1.4 as confluent_base
+FROM confluentinc/cp-base-new:latest as confluent_base
```

Comment:
This is dangerous to use in production.

Reply:
Agreed, however they are not publishing arm images for this container on the latest tag 6.1.9

Reply:
Thoughts on proceeding here ? They are publishing arm images with latest tag?

Reply:
Yes, the latest has an arm image. This is kafka-setup, doesn't impact quickstart. We could find or reverse engineer their Dockerfile as this confluent base image should only include open source components. Alternatively there may be some bitnami image to replace this. Long term this container should be merged into the system-update job.

Reply:
Can be pinned to the `manifest` instead of specific architectures' sha256. This was done as well as added some additional updates to the `kafka-setup` docker file to remove vulnerabilities.